### PR TITLE
feat: refactor the navigation-menu.blade.php

### PR DESCRIPTION
**Pull request by Ryuno-Ki**
This is not ready to be merged as there is no way to logout.

### CODE REVIEW

**Comment by Ryuno-Ki** [comment]
@Gurutosh Helf mir mal kurz auf die Sprünge: wie war das Logout angedacht?

**Comment by Ryuno-Ki** [comment]
Gestern besprochen: Figma wird angepasst, um Logout zu berücksichtigen.

File: resources/views/navigation-menu.blade.php [hunk 1]
```diff
@@ -10,209 +10,7 @@
...
-
-                            <!-- Authentication -->
-                            <form method="POST" action="{{ route('logout') }}" x-data>
-                                @csrf
-
-                                <x-dropdown-link href="{{ route('logout') }}"
-                                         @click.prevent="$root.submit();">
-                                    {{ __('Log Out') }}
-                                </x-dropdown-link>
-                            </form>
```
Comment (stevenobird) [inline resources/views/navigation-menu.blade.php:121]: We need that form with the link in some way to make sure users can logout.

Reason behind it being a POST seems to have to do with prefetching links, so using that method is a bit more bulletproof to ensure users don't get logged out by accident.

**Review by stevenobird — DISMISSED** [review]
See comment

File: resources/views/navigation-menu.blade.php (hunk 1 above)
Comment (Ryuno-Ki) [reply resources/views/navigation-menu.blade.php:121]: Allow me to quote myself:

> This is not ready to be merged as there is no way to logout.

The question is not whether but where to put the button.

Comment (Ryuno-Ki) [reply resources/views/navigation-menu.blade.php:121]: I was told that next to the menu buttons were good.

**Review by stevenobird — APPROVED** [review]

**Pull request merged**